test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1203,13 +1168,10 @@ def capture_comms(self, with_comm_logs=False):
         """
         self.logger.info("capturing comm information")
 
-        if len(self.all_hosts) == 0:
-            self.all_hosts.append(self.server_host)
-
-        # Go through the list of all hosts specified and collect comm info
-        for host in self.all_hosts:
+        # Capture comm logs
+        if self.comm_info_avail:
             if self.num_daemon_logs > 0 and with_comm_logs:
-                self.__capture_comm_logs(host)
+                self.__capture_comm_logs()
```

Comment:
Just an idle thought: What happens if the comm drops a core?  Should we grab a stack trace like we do for other daemons?